lib/exec.js

```diff
@@ -179,29 +180,43 @@ function sandboxNsjail(command, args, options) {
 
     for (const key of Object.keys(options.additionalEnv || {})) {
         jailingOptions.push(`--env=${key}=${options.additionalEnv[key]}`);
-        delete options.additionalEnv[key];
     }
+    delete options.additionalEnv;
+    return {
+        args: jailingOptions.concat(['--', command]).concat(args),
+        options,
+        filenameTransform,
+    };
+}
 
+function sandboxNsjail(command, args, options) {
+    logger.info('Sandbox execution via nsjail', {command, args});
+    const execPath = path.dirname(command);
+    const execName = path.basename(command);
+    options = options || {};
```

Comment:
this should be `options = {...options, customCwd: execPath}` to avoid modifying the options object being passed into `sandboxNsjail`

Reply:
Done, thanks.